feature/SOF-7165: Cypress tests for Standata imports

VsevolodX opened the pull request.

VsevolodX committed:
chore: import standata from NPM

VsevolodX committed:
chore: import standata

VsevolodX committed:
feat: add import from standata feature

VsevolodX committed:
update: create standata dialog widget

VsevolodX committed:
update: use standata dialog widget

VsevolodX committed:
update: add steps definitions

VsevolodX committed:
chore: adjust graphene fixture to not have residual properties

VsevolodX committed:
chore: update usages of standata import dialog

VsevolodX committed:
chore: imprt standata as devdep

VsevolodX committed:
chore: add package-lock.json

VsevolodX committed:
chore: remove noEmit from tsconfig

timurbazhirov commented on tests/cypress/fixtures/graphene.json:
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Graphene",
+    "name": "Graphene (mp-1040425)",
```
Can we take this from standata?

VsevolodX replied:
I think I left it this way since we're testing Standata itself.

VsevolodX committed:
Merge branch 'dev' into feature/SOF-7165

VsevolodX committed:
update: import standata in tests

VsevolodX committed:
try: add stadata path

VsevolodX committed:
chore: REMOVE noEmit thing

VsevolodX committed:
feat: import standata in utils

VsevolodX committed:
fix: a typo in name

VsevolodX committed:
update: read material from standata

VsevolodX committed:
chore: add mocha types

VsevolodX committed:
chore: restore graphene fixture for another test

switch to using only Standata should be done in a centralized manner in a separate task
